components/function-controller/deploy/jobinit/Dockerfile

```diff
@@ -25,16 +25,15 @@ RUN go build -ldflags "-s -w" -a -o jobinit cmd/jobinit/main.go \
 && mv ./jobinit /app/jobinit \
 && if [ -f ${BASE_APP_DIR}/licenses ]; then mv ${BASE_APP_DIR}/licenses /app/licenses; fi
 
-# get latest CA certs
-FROM alpine:latest as certs
-RUN apk --update add ca-certificates
-
 # result container
-FROM scratch
+FROM alpine:3.11
```

Comment:
The alpine 3.12 is, it would be good to bump the version

Reply:
Why is `alpine` better than `scratch`?

Reply:
We needed ssh in the container.